manager/integration/deploy/test.yaml

```diff
@@ -11,7 +11,8 @@ metadata:
 rules:
 - apiGroups: [""]
   resources: ["nodes", "nodes/status", "pods", "pods/exec", "persistentvolumes", "persistentvolumeclaims",
-              "persistentvolumeclaims/status", "secrets", "services", "serviceaccounts", "namespaces", "configmaps"]
+              "persistentvolumeclaims/status", "secrets", "services", "serviceaccounts", "namespaces", "configmaps",
```

Comment:
As I mentioned in another review, we can just use cluster admin role, since this is for testing.